community/modules/compute/SchedMD-slurm-on-gcp-partition/outputs.tf

```diff
@@ -26,7 +21,7 @@ output "partition" {
     static_node_count : var.static_node_count
     max_node_count : var.max_node_count
     zone : var.zone
-    image : data.google_compute_image.compute_image.self_link
+    image : "projects/${var.instance_image.project}/global/images/family/${var.instance_image.family}"
```

Comment:
what if the image does not have a family? did you try that? 
Are we unable to specify an image directly?

Reply:
I think the fix as is already brings value. I will approve it now and follow up on another task.

Reply:
SG I will experiment to bring back non-family image usage in a separate pr.